x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -10,18 +10,33 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const pageObjects = getPageObjects(['common', 'apiKeys']);
   const log = getService('log');
+  const security = getService('security');
+  const testSubjects = getService('testSubjects');
 
   describe('Home page', function() {
     this.tags('smoke');
     before(async () => {
+      await security.testUser.setRoles(['kibana_admin']);
       await pageObjects.common.navigateToApp('apiKeys');
     });
 
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    // https://www.elastic.co/guide/en/kibana/7.6/api-keys.html#api-keys-security-privileges
+    it('Shows required privileges ', async () => {
+      log.debug('Checking for section header');
+      const messageElement = await testSubjects.find('apiKeysPermissionDeniedMessage');
+      const message = await messageElement.getVisibleText();
+      expect(message).to.be('You need permission to manage API keys');
```

Comment:
Let's move the `await testSubjects.find('apiKeysPermissionDeniedMessage');` and `getVisibleText()` calls into a new page Objecto method like getPrivilegeMessage() that returns the visible text.

Reply:
adopted the suggestion